vars/neoDeploy.groovy

```diff
@@ -1,105 +1,68 @@
-import static com.sap.piper.Prerequisites.checkScript
-
 import com.sap.piper.ConfigurationHelper
 import com.sap.piper.Utils
-
 import com.sap.piper.tools.ToolDescriptor
-
+import com.sap.piper.tools.neo.DeployMode
+import com.sap.piper.tools.neo.NeoCommandHelper
+import com.sap.piper.tools.neo.WarAction
 import groovy.transform.Field
 
+import static com.sap.piper.Prerequisites.checkScript
+
 @Field String STEP_NAME = getClass().getName()
-@Field Set GENERAL_CONFIG_KEYS = []
-@Field Set STEP_CONFIG_KEYS = [
-    'account',
+@Field Set GENERAL_CONFIG_KEYS = [
+    'neo'
+]
+@Field Set STEP_CONFIG_KEYS = GENERAL_CONFIG_KEYS.plus([
     'dockerEnvVars',
     'dockerImage',
     'dockerOptions',
-    'host',
-    'neoCredentialsId',
     'neoHome'
-]
+])
+
 @Field Set PARAMETER_KEYS = STEP_CONFIG_KEYS.plus([
-    'applicationName',
-    'archivePath',
-    'deployAccount', //deprecated, replaced by parameter 'account'
-    'deployHost', //deprecated, replaced by parameter 'host'
+    'source',
```

Comment:
Is it intended that `deployMode`, `source` and `warAction` is **not** configurable via the config file?